File: GradientBackground/ViewController.m
//
//  ViewController.m
//  GradientBackground
//
//  Created by Pawan kumar on 9/18/17.
//  Copyright © 2017 Pawan Kumar. All rights reserved.
//

#import "ViewController.h"
#import "BackgroundLayer.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    //Add gradient background
    CAGradientLayer *bgLayer = [BackgroundLayer blueGradient];
    bgLayer.frame = self.view.bounds;
    [self.view.layer insertSublayer:bgLayer atIndex:0];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: Gradient-Background-Swift-5/Gradient-Background-Swift-5/ViewController.swift
//
//  ViewController.swift
//  Gradient-Background-Swift-5
//
//  Created by Pawan kumar on 20/07/19.
//  Copyright © 2019 Pawan Kumar. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet weak var imageView: UIImageView!

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        let backgroundLayer = BackgroundLayer.init()
        let layer = backgroundLayer.blueNewGradient(frame: self.view.bounds)
        //self.view.layer.insertSublayer(layer, at: 0)
        imageView.layer.insertSublayer(layer, at: 0)
    }


}

